apps/fluent-tester/src/FluentTester/TestComponents/Common/Slider.tsx

```diff
@@ -121,7 +121,7 @@ export const Slider: React.FunctionComponent<ISliderProps> = (props: ISliderProp
         setThumbLocation(initialThumbLocation);
       });
     }
-  }, [ref.current, initialValue, maximum, minimum]);
```

Comment:
Did eslint actually complain about this, or did you just notice it?

It makes sense to me to not memo on a ref, but just curious.

Reply:
Hm. I probably removed it because the linter said the useEffect wasn't depending on it, but now I see that it is. I'll take another look

Reply:
The linter did complain I think saying that the ref wasn't referenced in the hook, but now that I look, it does, so I'll have to take another look at this.

Reply:
I think the issue is that it can't depend on a property of an object, just an object, so I opted to pass in ref instead and it's fine with that.

Reply:
OK. FWIW, our sliders never worked to begin with, iirc.